.github/workflows/build.yml

```diff
@@ -191,15 +191,25 @@ jobs:
       - run: tar xzf artifacts.tgz
         working-directory: ${{ steps.tea.outputs.prefix }}
 
+      - run: |
```

Comment:
what does this step do tho? Output a file?

Reply:
Imports the gpg private key, so we can use it to sign.

Reply:
oh right. GPG is “you must have a daemon”. Half the reason everyone hates it.